app/renderer/css/main.css

```diff
@@ -273,6 +270,25 @@ body {
     width: 100%;
 }
 
+/*Pseudo element for loading indicator*/
+#webviews-container:before {
```

Comment:
Can you use `::before` here?

`::` is for indicating its a pseudo element whereas `:` is for pseudo-classes, this was something new is CSS3, browsers support the old `:` before for backward compatibility reasons.